public/static/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -0,0 +1,90 @@
+# Setup a Google Drive DVC Remote
+
+Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
+[remote storage](/doc/command-reference/remote).
+
+## Create and Configure a **Google Project**
+
+Since Google Drive has tight API usage quotas, one-time creation and
+configuration of a _Google Project_ is required. The mandatory
+[OAuth](https://developers.google.com/identity/protocols/OAuth2) credentials,
+**client ID** and **client secret**, can be generated only once. Please share
+them with other DVC users in your team to grant them access to Google's API.
+
+1. Log into the [Google Cloud Platform](https://console.developers.google.com)
+   account.
+
+2. Create a **New Project** or select an available one.
+
+3. From the left sidebar menu select **APIs & Services** -> **Dashboard** and
+   click on **ENABLE APIS AND SERVICES** to type and search for the **Google
+   Drive API**, in order to enable it from the search results.
+
+   ![](/static/img/gdrive-enable-apis-and-services.png)
+
+4. From the left sidebar menu select **APIs & Services** -> **OAuth consent
+   screen**, select a desired **User Type**, and click on the **Create** button.
+   On the next screen enter an **Application name** and click the **Save**
+   button. (You will be able to come back to edit others settings later.)
+
+5. From the left sidebar menu select **APIs & Services** -> **Credentials**, and
+   click the **Create credentials** button. Select the **OAuth client ID** ->
+   **Other** and click **Create** to proceed with a default suggested **Name**
+   for your new **OAuth client ID**.
+
+   ![](/static/img/gdrive-create-credentials.png)
+
+6. The newly generated **client ID** and **client secret** should be shown to
+   you. Please use these values below:
+
+  ```dvc
+  $ dvc remote add -d myremote gdrive://root/path/to/dir
+  $ dvc remote modify myremote gdrive_client_id my_gdrive_client_id
+  $ dvc remote modify myremote gdrive_client_secret gdrive_client_secret
+  ```
+
+  > Please see the [URL format](#url-format-for-google-drive-remotes) section
+  > below to learn how to form `gdrive://...` remote URLs.
+
+⚠️ It is safe to share **client ID** and **client secret** at with your team.
+These credentials are used to
+[retrieve an authorization token](https://developers.google.com/identity/protocols/OAuth2)
+and
+[authenticate your application](https://developers.google.com/drive/api/v2/about-auth)
+with Google. Credentials do not provide access to all of your Google Drive
+storage. Note that Google rate-limits the global number of queries per each
+**client ID**, so uncontrolled usage of a **client ID** can lead to
+[reaching the allowed Google API usage limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#quota).
```

Comment:
@MaxRis @shcheklein @efiop what version of the Google Drive API does DVC use? The links here are to v2 docs but the newest is v3. Thanks

Reply:
v2 is the version we use (mostly because PyDrive - a python wrapper around API is using v2)

Reply:
OK I hope someone notices if/when PyDrive updates the GDrive API version and updates this guide's links. Unlikely 😋